doc/adr/0003/xy-output-component.tsx

```diff
@@ -0,0 +1,913 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Firstly, maybe we should rename it (prefix with proto_ for example). I can see someone accidentally editing this version instead of the file in the source code location.

I copied it over to the code, but it didn't compile. compilation errors are due the rename of `filtrer` to `filter`. The other one I didn't investigate further. Maybe for this commit, it should be compatible with the current code. I'm aware that in the future it will break, but it's ok.

Reply:
Changes implemented.